.github/workflows/ci.yml

```diff
@@ -8,37 +8,62 @@ jobs:
 
     strategy:
       matrix:
-        free-solid-svg-icons: [5.x, 6.x]
-        fontawesome-svg-core: [1.2.x, 6.x]
-        node-version: [18.x, 20.x, 22.x]
-        vue: [3.0.x, 3.1.x, 3.2.x, 3.3.x]
+        free-solid-svg-icons: [7.x, 6.x, 5.x]
+        fontawesome-svg-core: [7.x, 6.x, 1.2.x]
+        node-version: [20.x, 22.x]
+        vue: [3.0.x, 3.1.x, 3.2.x, 3.3.x, 3.4.x, 3.5.x]
+        exclude:
+          # Exclude Font Awesome 7.x free-solid-svg-icons with older svg-core versions
+          - free-solid-svg-icons: '7.x'
+            fontawesome-svg-core: '6.x'
+          - free-solid-svg-icons: '7.x'
+            fontawesome-svg-core: '1.2.x'
 
     steps:
       - uses: actions/checkout@v1
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: v5 install, build, and test
-        if: ${{ matrix.free-solid-svg-icons }} == '5.x'
+      - name: v7 install, build, and test
+        if: ${{ matrix.free-solid-svg-icons == '7.x' && matrix.fontawesome-svg-core == '7.x' && matrix.vue >= '3.2.x' }}
         run: |
-          npm install -g npm@8
+          npm install -g npm@10
           npm install
-          npm install --no-save @fortawesome/fontawesome-svg-core@1.2.x @fortawesome/free-solid-svg-icons@5.x vue@${{ matrix.vue }}
+          npm install --no-save ./fortawesome-fontawesome-svg-core-7.0.0-alpha8.tgz ./fortawesome-fontawesome-common-types-7.0.0-alpha8.tgz @fortawesome/free-solid-svg-icons@6.x vue@${{ matrix.vue }}
+          if [[ "${{ matrix.vue }}" < "3.2.13" ]]; then
+            npm install --no-save @vue/server-renderer@${{ matrix.vue }}
+          fi
```

Comment:
See this note here from vue/server-render:  https://github.com/vuejs/core/tree/main/packages/server-renderer#vueserver-renderer

> Note: as of 3.2.13+, this package is included as a dependency of the main vue package and can be accessed as vue/server-renderer. This means you no longer need to explicitly install this package and ensure its version match that of vue's. Just use the vue/server-renderer deep import instead.

From my understanding of this and the test failure errors that were occurring in our CI tests, vue versions 3.0.x and 3.1.x now need vue/server-renderer but it was not included as a dependency in vue version prior to 3.2.13.
So this step was needed to make the ci tests pass (dependency issues).